bin/find_delta_remote.js

```diff
@@ -0,0 +1,88 @@
+const fs = require('fs/promises')
+const { spawn, spawnSync } = require('child_process')
+const path = require('path')
+const {argv0} = require('process')
+
+/**
+ *
+ * @param {string[]} args arguments for the command
+ * @param {import('child_process').SpawnOptionsWithoutStdio} [options]
+ */
+async function run(command, args, options, listener = undefined) {
+  return new Promise((resolve, reject) => {
+    console.log(`- Executing "${command} ${args.join(' ')}"`)
+    p.stdout.pipe(process.stdout)
+  })
+}
```

Comment:
whats this used for?

Reply:
That's dead code, i first copied the run function from another of our scripts

Reply:
Oh that's dead code, will remove it